qvariantlessthan.h

```diff
@@ -0,0 +1,12 @@
+#ifndef QVARIANTLESSTHAN_H
+#define QVARIANTLESSTHAN_H
```

Comment:
pragma once?

Reply:
I would have to refactor the whole lib for consistency. Probably it's not worth to do in terms of convenience of merging upstream changes in the future.